src/services/storage.js

```diff
@@ -1,17 +1,8 @@
 //this service is made to handle all localstorage variables
-const NEWSLETTERKEY = "newsletter";
-let newsLetterValue = localStorage.getItem(NEWSLETTERKEY);
-
-export const getNewsletterSubcription = () => {
-  return newsLetterValue;
-};
-
-export const subscribeNewsletter = () => {
-  localStorage.setItem(NEWSLETTERKEY, true);
-  newsLetterValue = true;
+export const items = {
+  isSubscribedToNewsletter: "isSubscribedToNewsletter",
 };
 
-export const unsubscribeNewsletter = () => {
-  localStorage.removeItem(NEWSLETTERKEY);
-  newsLetterValue = false;
-};
+export const getItem = (key) => localStorage.getItem(key);
+export const setItem = (key, value) => localStorage.setItem(key, value);
```

Comment:
It would be great to export these methods as a default export (see [my earlier suggestion](https://github.com/sievins/author-page/pull/41#discussion_r609082494)). That way it's easy to distinguish between the methods that manipulate the local storage and the `items`.